pkg/controllers/management/drivers/nodedriver/machine_driver.go

```diff
@@ -52,6 +37,7 @@ var (
 const (
 	driverNameLabel  = "io.cattle.node_driver.name"
 	uiFieldHintsAnno = "io.cattle.nodedriver/ui-field-hints"
+	aliasedAnno      = "fileToFieldAliases"
```

Comment:
@kinarashah WDYT, should we add some sort of prefix to this?

Reply:
Good idea, tho the other keys for driver data right now don't have any prefix so I also don't want the annotations to look inconsistent? We would need to port the change for other fields `publicCredentialFields` and `privateCredentialFields` as well.